docusaurus.config.js

```diff
@@ -0,0 +1,134 @@
+  onBrokenMarkdownLinks: 'warn',
+  favicon: 'img/favicon.ico',
+  organizationName: 'graphql',
+  projectName: 'graphql-js',
+  themeConfig: {
+    navbar: {
+      title: 'graphql-js',
+      logo: {
+        alt: 'GraphQL Logo',
+        src: 'img/logo.svg',
+      },
+      items: [
+        {
+          type: 'doc',
+          docId: 'index',
+          position: 'left',
+          label: 'Tutorial',
+        },
+        {
+          href: 'https://github.com/graphql/graphql-js',
+          label: 'GitHub',
+          position: 'right',
+        },
+        {
+          to: 'api/graphql-js',
+          label: 'API',
+          position: 'left',
+        },
+      ],
+    },
+    footer: {
+      style: 'dark',
+      links: [
+        {
+          title: 'Docs',
+          items: [
+            {
+              label: 'Tutorial',
+              to: 'tutorials',
+            },
+          ],
+        },
+        {
+          title: 'Community',
+          items: [
+            {
+              label: 'Code of Conduct',
+              href: 'https://graphql.org/codeofconduct/',
+            },
+            {
+              label: 'Stack Overflow',
+              href: 'https://stackoverflow.com/questions/tagged/graphql-js',
+            },
+            {
+              label: 'Discord',
+              href: 'https://discord.graphql.org/',
+            },
+            {
+              label: 'Twitter',
+              href: 'https://twitter.com/graphql',
+            },
+            {
+              label: 'Upcoming Events',
+              href: 'https://graphql.org/community/upcoming-events/',
+            },
+          ],
+        },
+        {
+          title: 'More',
+          items: [
+            {
+              label: 'GraphQL JS Working Group',
+              href: 'https://github.com/graphql/graphql-js-wg',
+            },
+            {
+              label: 'GraphQL Foundation',
+              href: 'https://graphql.org/foundation',
+            },
+            {
+              label: 'GraphQL Spec',
+              href: 'https://spec.graphql.org',
+            },
+          ],
+        },
+      ],
+      copyright: `Copyright © ${new Date().getFullYear()} <a href="https://graphql.org/foundation/" target="_blank" rel="noopener noreferrer">The GraphQL Foundation</a>. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our <a href="https://www.linuxfoundation.org/trademark-usage" target="_blank" rel="noopener noreferrer">Trademark Usage</a> page. Linux is a registered trademark of Linus Torvalds. <a href="https://www.linuxfoundation.org/privacy" target="_blank" rel="noopener noreferrer">Privacy Policy</a> and <a href="https://www.linuxfoundation.org/terms" target="_blank" rel="noopener noreferrer">Terms of Use</a>. Built with <a href="https://docusaurus.io/" target="_blank" rel="noopener noreferrer">Docusaurus</a>.`,
```

Comment:
What place do you copy it from?
Ideally, we would use the same text as graphql.org
![image](https://user-images.githubusercontent.com/8336157/142279138-3d962332-2e36-4745-b344-95d1197edd04.png)

Reply:
When I copied long while ago that is what main site used to have but it got updated recently. https://github.com/graphql/graphql.github.io/commit/a1dde2742232febfb5741d6587342a077c3889c8